### PR TITLE
Mallets - Some small fixes

### PR DESCRIPTION
**Pull request by zonkmachine**
Found an oddity caused by the Random function. Fixed up some older oddities and glitches at the same time.

* Fix an issue with the Random algorithm where the randomized variables ended up half as big as they should be.
* Tubular Bells
  * Implement the ADSR knob which had no action. The value needed updating.
  * LFO - update values while playing.
* BandedWG (Uniform Bar, Tuned Bar, etc.) - Randomize values of Pressure and Speed.
* ModalBar (Marimba, Vibraphone, etc.) - Update values of Vibrato (Gain, Frequency) while playing.

The change on the Vibrato is how you expect the knobs to work. On a long note you should hear the vibrato change when you turn the knobs but it was only updated on every new note. There is no way to create an upgrade routine for this but the effect should be subtle and not cause any problems. Same with the ADSR. The knob didn't have an effect previously but if it was turned, with no perceived effect, the new value was still saved with the project correctly so in this case older projects may not upgrade properly. This will also be subtle changes where the ADSR knob modulates the action of the Modulator and Crossfade knobs. You could probably dig into the STK code here but I don't think it's going to be worth the effort. Been there, done that.

The ModalBar Vibrato and stick mix control messages in stk seem to have been mixed up. Reported upstream.
https://github.com/thestk/stk/issues/143

### CODE REVIEW

File: plugins/Stk/Mallets/Mallets.cpp [hunk 1]
```diff
@@ -369,7 +379,13 @@ void MalletsInstrument::playNote( NotePlayHandle * _n,
 	const f_cnt_t offset = _n->noteOffset();
 
 	auto ps = static_cast<MalletsSynth*>(_n->m_pluginData);
-	ps->setFrequency( freq );
+	ps->setFrequency(freq);
+	if (p==9) // Tubular Bells
```
Comment (sakertooth) [inline plugins/Stk/Mallets/Mallets.cpp:383]:
```suggestion
	if (p == 9) // Tubular Bells
```

Comment (zonkmachine) [reply plugins/Stk/Mallets/Mallets.cpp]: fixed in the latest update

**Comment by zonkmachine** [comment]
I had a bit of a struggle with the code to update values for the Vibrato in the ModalBar (Marimba - Clump). There seem to be an issue in stk where the control values have been mixed up so when I tried to update the vibrato the direct stick mix got turned up instead...

**Comment by zonkmachine** [comment]
I believe this is ready to merge. Please note that we intentionally use the wrong control value for Vibrato Frequency but this is nothing new. The control messages for Vibrato Frequency and Stick Mix are already the wrong values in Mallets and this is due to an upstream issue.

**Review by DomClark — APPROVED** [review]

**Comment by zonkmachine** [comment]
> The ModalBar Vibrato and stick mix control messages in stk seem to have been mixed up. Reported upstream.
> [thestk/stk#143](https://github.com/thestk/stk/issues/143)

Fixed upstream in https://github.com/thestk/stk/pull/144

@DomClark Is there a way to probe the version of stk? I made a workaround in this PR for this issue but as it's fixed upstream, by me, this workaround will have to be reverted or be made to depend on the stk version.

**Pull request merged**